.mailer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+require '.db_config.php';
+require_once '.functions.php';
+
+// Data array
+$data = array();
+
+// Verification logic
+$verifiedTo = isset($_POST['to']) && $_POST['to'] != '';
+$verifiedSubject = isset($_POST['subject']) && $_POST['subject'] != '';
+$verifiedBody = isset($_POST['body']) && $_POST['body'] != '';
+
+if (!($verifiedTo && $verifiedSubject && $verifiedBody && $_POST['sessionId'])) {
+    $data['success'] = false;
+    echo (json_encode($data));
+    return;
+}
+
+$connection = openDatabaseConnection();
+
+if (getUser($_POST['sessionId'], $connection)['admin'] == 0) {
+    $data['success'] = false;
+    echo (json_encode($data));
+    $connection = null;
+    return;
+}
+
+$connection = null;
+
+$url = 'http://lp13.rpiambulance.com:3000/sendmail';
```

Comment:
This needs to be something like `mailer.lp13.rpiambulance.com`

Reply:
And probably should be `https`

Reply:
Updated